node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeFailer.java

```diff
@@ -169,27 +170,38 @@ private boolean failAllowedFor(NodeType nodeType) {
     }
 
     /**
-     * If the node is positively DOWN, and there is no "down" history record, we add it.
-     * If the node is positively UP we remove any "down" history record.
+     * If a node remains bad for a long time, the NodeFailer will eventually try to fail the node.
```

Comment:
I think this JavaDoc should explain what this method returns instead,

Reply:
Added a line for that.